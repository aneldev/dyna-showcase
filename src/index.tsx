import * as React from 'react';
import {DynaReactComponentShowcase} from './DynaReactComponentShowcase';
import {faIcon} from './utils/faIcon';

const menuStyleWhite = require('../styles/menu-style-white.less');
const menuStyleRed = require('../styles/menu-style-red.less');

import {IShowcase, IShowcaseView, IShowcaseViewProps } from './interfaces';

import "font-awesome/css/font-awesome.css";
import "./index.less";

export{
  IShowcase, IShowcaseView, IShowcaseViewProps,
  DynaReactComponentShowcase,
  faIcon,
  menuStyleWhite, menuStyleRed,
}
