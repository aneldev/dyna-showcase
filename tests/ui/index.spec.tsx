import * as React from 'react';
import * as enzyme from 'enzyme';

import {DynaReactComponentShowcase} from './../../src';

describe('Home', () => {
  let wrapper;

  it('has expected content with deep render', () => {
    wrapper = enzyme.shallow(
      <DynaReactComponentShowcase />,
      {}
    );

    expect(wrapper).toMatchSnapshot();
  });
});
