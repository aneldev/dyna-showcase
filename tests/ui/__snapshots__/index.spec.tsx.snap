// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home has expected content with deep render 1`] = `
<BrowserRouter>
  <div
    className="appRouter"
  >
    <Route
      render={[Function]}
    />
    <Route
      path="/:viewSlug?/:propsSlug?"
      render={[Function]}
    />
  </div>
</BrowserRouter>
`;
